build.sbt

```diff
@@ -19,7 +19,7 @@ resolvers ++= Seq(
   "oss releases" at "http://oss.sonatype.org/content/repositories/releases"
 )
 
-releaseSettings
+releaseCrossBuild := true
```

Comment:
You may be correct: in a fit of confusion because the `releaseSettings` setting is no longer supported I added something.

I'm confused though, I thought we did need to cross-compile for Scala 2.10 and 2.11?

:+1: on creating this project BTW